Police-Data-Accessibility-Project/data-sources-app-v2
# Mc 473 input output schema consolidation

### Fixes

* https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/475
* https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/473

### Description

* Consolidates input and output schemas into an endpoint schema configuration, which is referenced by integration tests, endpoint input validation and population logic, documentation.
* This was done to ensure a single source of truth in all of these locations
* Some logic was updated to correct for bugs discovered in this consolidation process.
* Add `request_urgency` to `data_requests` table
* LIBTYFI: Add test to ensure that API documentation loads properly when called.

### Testing

* Run tests in `tests` directory
* Check api documentation to ensure it conforms with expected logic.

### Performance

* Performance impact marginal.

### Docs

* API Docs updated

### Breaking Changes

* No breaking changes expected.
